optee-packages.mk

```diff
@@ -0,0 +1,53 @@
+OPTEE_PLATFORM ?= poplar
+OPTEE_PLATFORM_FLAVOR ?= poplar
+OPTEE_CFG_ARM64_CORE ?= y
+OPTEE_TA_TARGETS ?= ta_arm32
+OPTEE_OS_DIR ?= optee/optee_os
```

Comment:
We need to add this to Android build tree first, right? What is the source for that?
Is following understanding correct?
git clone source_of_optee ${ANDROID_BUILD_TOP}/optee/optee_os

Reply:
I'm using the swg local manifest in my builds wget https://raw.githubusercontent.com/linaro-swg/optee_android_manifest/lcr-ref-hikey-o/swg.xml